dashboard/main.go

```diff
@@ -62,7 +65,7 @@ func main() {
 	gosms.InitWorker(modems, bufferSize, bufferLow, loaderTimeout, loaderCountout, loaderTimeoutLong)
 
 	log.Println("main: Initializing server")
-	err = InitServer(serverhost, serverport)
+	err = InitServer(serverhost, serverport, serverusername, serverpassword)
```

Comment:
Nothing, web server will be open as default (conf.ini has USERNAME and PASSWORD commented out)